extensions/wikia/CategoryPagination/templates/category.tmpl.php

```diff
@@ -0,0 +1,36 @@
+<? if ( $empty ): ?>
+	<?= wfMessage( 'category-empty' )->parse(); ?>
+<? else: ?>
+	<?= $categoryGallery; ?>
+
+	<div lang="<?= htmlspecialchars( $lang->getCode() ); ?> dir="<?= htmlspecialchars( $lang->getDir() ); ?>">
+	<? if ( !empty( $content['subcat'] ) ): ?>
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "subcat" 5 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)